_pages/dev/styling-and-page-layout/skeleton-design.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Skeleton Design
+feature:
+- name: Skeleton Design
+  spa_version: 3.0
+  cx_version: 1905
```

Comment:
There's actually no relation to the backend with this feature. Can we leave that our?

Reply:
This data is used to populate a "feature release versions" table here: https://sap.github.io/spartacus-docs/release-information/#feature-release-versions

I had a bit of a discussion with our PO about this. He pointed out that there were certain features that didn't depend on the back end, which I assumed was the case with the skeleton design feature. But we want people working with Spartacus to use "at least" version 1905 (not earlier). So that is where this value comes in. In many cases, it ends up being the equivalent of "not applicable" -- but I didn't want to put "not applicable in the table in case people get the idea they can use a version that is older than 1905 😄 

Let me know if you are ok with this 🙂

Reply:
I prefer "Any version" in this case ;)

Reply:
Done 😄